dataprofiler/tests/profilers/test_base_column_profilers.py

```diff
@@ -284,9 +281,6 @@ def test_updated_profile(self):
 
         for profiler_mock in profiler_mocks:
             self.assertEqual(1, profiler_mock.call_count)
-            self.assertEqual(
-                2, profiler_mock.return_value.update.call_count
-            )
```

Comment:
Can't keep a count of something which isn't called (mock the profiler) and mock's can't be pickled so this errors in multiprocess mode.